test_data/workflow_api.json

```diff
@@ -0,0 +1,138 @@
+{
+    "3": {
+        "inputs": {
+            "seed": 3254890760919,
+            "steps": 20,
+            "cfg": 8,
+            "sampler_name": "euler",
+            "scheduler": "normal",
+            "denoise": 0.5,
+            "model": [
+                "4",
+                0
+            ],
+            "positive": [
+                "6",
+                0
+            ],
+            "negative": [
+                "7",
+                0
+            ],
+            "latent_image": [
+                "11",
+                0
+            ]
+        },
+        "class_type": "KSampler",
+        "_meta": {
+            "title": "KSampler"
+        }
+    },
+    "4": {
+        "inputs": {
+            "ckpt_name": "realisticVisionV60B1_v51HyperVAE.safetensors"
```

Comment:
I think vanilla sd15 checkpoint will do better here because it is even more default. Still, I am ok to merge like this, will add some notes in the readme about it later

Reply:
I am fine with both, it is just that personnaly I have less trouble finding this checkpoint that the original SD1.5